piccolo3/server/PiccoloConfig.py

```diff
@@ -51,9 +51,18 @@
 
 [coolbox]
   update_interval = integer(default=5) # update inteval in sec
+  serial_location = string(default="/dev/ttyUSB0")
+  coolbox_log_path = string(default="./coolbox.log")
   [[temperature_sensors]]
      [[[__many__]]]
-       target = float(default=10.) # target temperature
+       target = float(default=20.) # target temperature
+  [[fans]]
+    [[[__many__]]]
+      fan_on = boolean(deafult=False)
+  [[voltage_sensors]]
+    [[[__many__]]]
+  [[current_sensors]]
+    [[[__many__]]]
```

Comment:
not sure if you need to add these since there are no configurable options. I would just hard code them into the control class

Reply:
Makes sense, my thinking was that they might have needed to be present in the config to trigger something elsewhere in the code. As unnecessary, I've removed them.